### DISCUSSION TITLE
Who won both the 🎥🏆 and Nobel🏅?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#847] [Galaxy Trivia] [answered]
**Bonus:** For what accomplishments?

**Comment by peterbaccaro** [comment, marked as answer]
The only two people to have won an Oscar and a Nobel Prize are **Bob Dylan** and **George Bernard Shaw**.

**Bob Dylan**
- In 2016, he was awarded the Nobel Prize in Literature "for creating new poetic expressions within the great American song tradition".
- In 2001 he received an Oscar for Best Original Song for his song "Things Have Changed," which was featured in the film "Wonder Boys".

**George Bernard Shaw**
- In 1925, he was awarded the Nobel Prize in Literature for his contributions to literature and his mastery of dramatic art.
- In 1939, he was awarded an Oscar for Best Writing, Screenplay for his role in adapting his own play Pygmalion for the screen. Pygmalion would go on to be remade as the wildly successful musical and film My Fair Lady.

**Reply by ParanoidUser** [reply]
Is he the only one? 🤔

**Reply by peterbaccaro** [reply]
**George Bernard Shaw** also, I have update above.
